src/workloads/serverless/IndexStress.yml

```diff
@@ -0,0 +1,269 @@
+    - OperationName: updateOne
+      OperationCommand:
+        Filter: {id: {^RandomInt: {min: 0, max: 1000}}}
+        Update:
+          $inc: {a: 1}
+
+- Name: Loader3
+  Type: Loader
+  ClientName: Client3
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: largeins
+    CollectionCount: 1
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: 1000
+    Indexes:
+    - keys: {hash: 1, key: 1, string1: 1, string2: 1}
+    - keys: {key: 1}
+    Document:
+      _id: {^Inc: {start: 0}}
+      hash: &randomString3 {^RandomString: {length: 22}}
+      key: *randomString3
+  - {Nop: true}
+
+- Name: FindLargeIndex3
+  Type: CrudActor
+  Threads: 32
+  Database: largeins
+  ClientName: Client3
+  Phases:
+  - {Nop: true}
+  - Name: FindLargeIn
+    Duration: 30 minutes
+    GlobalRate: 1000 per 15 seconds
+    Collection: Collection0
+    Operations:
+    - OperationName: find
+      OperationCommand:
+        Filter:
+          hash:
+            $in: &arrayGenerator3 {^FixedGeneratedValue: {fromGenerator: {^Array: {of: *randomString3, number: 20000}}}}
+          key:
+            $in: *arrayGenerator3
+
+- Name: Update3
+  Type: CrudActor
+  Database: largeins
+  ClientName: Client3
+  Threads: 10
+  Phases:
+  - {Nop: true}
+  - Duration: 30 minutes
+    RecordFailure: true
+    GlobalRate: 1 per 250 milliseconds
+    Collection: Collection0
+    Operations:
+    - OperationName: updateOne
+      OperationCommand:
+        Filter: {id: {^RandomInt: {min: 0, max: 1000}}}
+        Update:
+          $inc: {a: 1}
+
+- Name: Loader4
+  Type: Loader
+  ClientName: Client4
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: largeins
+    CollectionCount: 1
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: 1000
+    Indexes:
+    - keys: {hash: 1, key: 1, string1: 1, string2: 1}
+    - keys: {key: 1}
+    Document:
+      _id: {^Inc: {start: 0}}
+      hash: &randomString4 {^RandomString: {length: 22}}
+      key: *randomString4
+  - {Nop: true}
+
+- Name: FindLargeIndex4
+  Type: CrudActor
+  Threads: 32
+  Database: largeins
+  ClientName: Client4
+  Phases:
+  - {Nop: true}
+  - Name: FindLargeIn
+    Duration: 30 minutes
+    GlobalRate: 1000 per 15 seconds
+    Collection: Collection0
+    Operations:
+    - OperationName: find
+      OperationCommand:
+        Filter:
+          hash:
+            $in: &arrayGenerator4 {^FixedGeneratedValue: {fromGenerator: {^Array: {of: *randomString4, number: 20000}}}}
+          key:
+            $in: *arrayGenerator4
+
+- Name: Update4
+  Type: CrudActor
+  Database: largeins
+  ClientName: Client4
+  Threads: 10
+  Phases:
+  - {Nop: true}
+  - Duration: 30 minutes
+    RecordFailure: true
+    GlobalRate: 1 per 250 milliseconds
+    Collection: Collection0
+    Operations:
+    - OperationName: updateOne
+      OperationCommand:
+        Filter: {id: {^RandomInt: {min: 0, max: 1000}}}
+        Update:
+          $inc: {a: 1}
+
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq: replica
```

Comment:
This will run for normal replica sets in sys-perf as well. Is that okay, considering the URIs wouldn't be overridden for them?

Reply:
Ah, thank you for pointing this out! Siyuan and I chatted about whether or not we wanted to use the AutoRun feature for our tests, and we're not ready to automatically schedule them to run yet. I'm just going to remove this section.